types/rlp_unmarshal_storage.go

```diff
@@ -30,12 +31,25 @@ func (b *Body) UnmarshalRLPFrom(p *fastrlp.Parser, v *fastrlp.Value) error {
 		return err
 	}
 
-	for _, txn := range txns {
+	for i := 0; i < len(txns); i++ {
```

Comment:
Duplicate code? https://github.com/0xPolygon/polygon-edge/pull/1114/files#diff-efc5999399a6ff1082b29a52ce57fd7bfd6ebe4789d686a44c70523d2a61c684R83

Reply:
They are similar but use different objects. Generalising this func would look overcomplicated IMO. Feel free to suggest something if you want.